common/migration_mechanism.py

```diff
@@ -106,6 +113,7 @@ def run_migrations(data_root_path: str) -> None:
     :param data_root_path: the root path of data files to migrate
     :return: None
     """
+    _ensure_account_database_exists(data_root_path)
```

Comment:
Please move this down after the backup so that the backup is identical to the starting state of the directory.